doc/Language/containers.rakudoc

```diff
@@ -239,8 +239,10 @@ lexpad at runtime.
 
 The answer is that binding to array elements is recognized at the syntax
 level and instead of emitting code for a normal binding operation, a special
-method (called L<C<BIND-KEY>|/routine/BIND-KEY>) is called on the
-array. This method handles binding to array elements.
+method (called L<C<BIND-POS>|/routine/BIND-POS>) is called on the
+array. This method handles binding to array elements. There is also an
```

Comment:
I tripped over this sentence a bit:

> This method handles binding to array elements. There is also an equivalent method for binding to hash elements as well (called L<C<BIND-KEY>|/routine/BIND-KEY>)

I can explain why if you want me to (just ask) but for now I'll simply suggest what would avoid me tripping up):

> (C<BIND-POS>handles binding to array elements. There is also a method for binding to I<hash> elements called L<C<BIND-KEY>|/routine/BIND-KEY>.)

Reply:
Sigh. The suggestion I wrote began with "`C<BIND-POS>` handles..." and ended with a proper RakuDoc link but of course GH gobbles such text. I'm tempted to delete my review given that it was just a very small thing but please just guess what I meant or feel free to ignore my comment.